internal/commands/executor.go

```diff
@@ -96,10 +121,11 @@ func (e executorImpl) All() internal.AllServices {
 }
 
 // NewExecutor creates the default Executor
-func NewExecutor(cfg *config.Config, svc internal.AllServices) Executor {
+func NewExecutor(cfg *config.Config, svc internal.AllServices, logger flume.Logger) Executor {
```

Comment:
Added the logger to the executor, rather than changing the execute() signatures all over. Not sure if this is the best option?